Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -302,6 +301,64 @@
       - CableHVStack
       - TimerTrigger
 
+- type: entity
+  parent: Protolathe
+  id: MedicalTechFab
+  name: Medical TechFab
```

Comment:
```suggestion
  name: medical techfab
```

Reply:
noticing that a lot of these are uppercased actually so if you wanna go through and change those you can